Dockerfile

```diff
@@ -119,18 +141,18 @@ RUN \
   docker-compose \
```

Comment:
Are `docker-cli` and `docker-compose` necessary within the container?

Reply:
Yes. We're using them for both cloud function and the installation command. We might split them into separate dockerfiles in the future, but for now it's easier to maintain as one.